README.md

```diff
@@ -3,28 +3,31 @@
 # Responsible AI Toolbox
 Responsible AI is an approach to assessing, developing, and deploying AI systems in a safe, trustworthy, and ethical manner, and take responsible decisions and actions.
 
-Responsible AI Toolbox is a suite of tools provides a collection of model and data exploration and assessment user interfaces that enable a better understanding of AI systems. These interfaces empower developers and stakeholders of AI systems to develop and monitor AI more responsibly, and take better data-driven actions.
+Responsible AI Toolbox is a suite of tools providing a collection of model and data exploration and assessment user interfaces and libraries that enable a better understanding of AI systems. These interfaces and libraries empower developers and stakeholders of AI systems to develop and monitor AI more responsibly, and take better data-driven actions.
 
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/microsoft/responsible-ai-widgets/main/img/responsible-ai-toolbox.png" alt="Model Assessment" width="250"/>
+<img src="./img/responsible-ai-toolbox.png" alt="ResponsibleAIToolboxOverview" width="750"/>
```

Comment:
I'm not sure if this works in all contexts, like on the pypi release page (see https://pypi.org/project/raiwidgets/)?  Why did you decide to change it to relative path instead of website path?